### DISCUSSION TITLE
What is your favorite character in BombSquad?

### DISCUSSION THREAD

**Discussion by HeyErfan** [efroemling/ballistica#681] [Polls]
Sorry, Due to the limitation of adding more than 8 options in the poll, I couldn't assign each character to only one option. Therefore, I decided to put the characters that I think were less used by people in one option.

**Comment by Se7enCodes** [comment]
There is a new character, which was released a few weeks ago I guess. She is Betty, an old granny. I really like her, due to her body, and her sound effects. Very funny, and intuitive??
